test/runtests.jl

```diff
@@ -2,4 +2,14 @@ using Libm
 using Base.Test
 
 # write your own tests here
-@test 1 == 2
+
+x = 2*rand(10000,1)-1
```

Comment:
Yeah, this was just an initial hack to get in ported and test timings. Just to check it agrees in an arbitrary range.

If we decide forward with this implementation I can add more robust tests.
What do you think? Is this something we want to keep? 
There is also  https://github.com/shibatch/sleef  implementations.